src/components/styles/Layout.tsx

```diff
@@ -21,13 +21,14 @@ export const PageLayout = styled(Layout)`
   ${whiteBackground}
 `;
 
-const siderWidth = "275px";
```

Comment:
nit: it could be easier to organize the styles if we keep this variable and use it for lines 27-31